posthog/hogql/functions/mapping.py

```diff
@@ -462,7 +462,7 @@ def compare_types(arg_types: list[ConstantType], sig_arg_types: tuple[ConstantTy
     "age": HogQLFunctionMeta("age", 3, 3),
     "dateDiff": HogQLFunctionMeta("dateDiff", 3, 3),
     "dateTrunc": HogQLFunctionMeta("dateTrunc", 2, 2),
-    "dateAdd": HogQLFunctionMeta("dateAdd", 3, 3),
```

Comment:
`dateAdd` is an alias if `date_add` - so I don't see why this wouldn't work. [docs](https://clickhouse.com/docs/en/sql-reference/functions/date-time-functions#date_add). I'd be keen to not change this if we can help it, it'll become one of the only funcs in snake case, and may break existing queries

Reply:
<img width="416" alt="image" src="https://github.com/PostHog/posthog/assets/1459269/b6351235-bbd8-4111-8bfa-f1c626c6947e">